python/ray/autoscaler/aws/node_provider.py

```diff
@@ -121,9 +126,13 @@ def create_node(self, node_config, tags, count):
                 "Key": k,
                 "Value": v,
             })
+        subnet_ids = conf.pop('SubnetIds')
```

Comment:
comment here that SubnetIds is not a real config key, and we need to resolve it before handing it to AWS

Reply:
agreed, comment added